rpcs3/emucore.vcxproj.filters

```diff
@@ -1991,6 +1991,21 @@
       <Filter>Emu</Filter>
     </ClInclude>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="Emu\RSX\Common\Interpreter\FragmentInterpreter.glsl">
+    <ClInclude Include="Emu\Audio\audio_device_listener.h">

```

Comment:
I think this is already present in the filters

Reply:
Indeed, this is likely an issue with msvc editing the files + rebasing the branch. Will fix.